docs/howto/operator/gcp/kubernetes_engine.rst

```diff
@@ -0,0 +1,122 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Google Kubernetes Engine Operators
```

Comment:
Can you add link to this guide in the Kubernetes Pod Operator guide? 
https://airflow.readthedocs.io/en/latest/howto/operator/kubernetes.html

```
    .. note::
        If you use `Google Kubernetes Engine <https://cloud.google.com/kubernetes-engine/>`__, use
        :ref:`GKEStartPodOperator <howto/operator:GKEStartPodOperator>`__ operaator, which
        simplifies the authorization process.
```

Reply:
[Done here](https://github.com/apache/airflow/pull/8883/commits/3fe1780ae0f9503988c725c6b12b2bdf9b8e6bed)

Reply:
[Follow up commit - had to correct the format](https://github.com/apache/airflow/pull/8883/commits/ded5cca846c7f39bccb707046e31770a7578064a)